api/src/main/java/org/commonjava/indy/conf/SSLStoreValidationConfig.java

```diff
@@ -0,0 +1,47 @@
+package org.commonjava.indy.conf;
+
+
+import org.commonjava.propulsor.config.annotation.ConfigName;
+import org.commonjava.propulsor.config.annotation.SectionName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.File;
+import java.io.InputStream;
+
+@ApplicationScoped
+@SectionName( SSLStoreValidationConfig.SECTION_NAME )
+public class SSLStoreValidationConfig implements IndyConfigInfo {
```

Comment:
We already talked about this in person, but I like the idea of making this a general-purpose internal feature-flag configuration mechanism. I think renaming this to something like `InternalFeatureConfig` (with a similar change to the section-name) would make sense.